common/types/mapper/proto/history_test.go

```diff
@@ -153,7 +155,11 @@ func TestHistoryQueryWorkflowRequest(t *testing.T) {
 }
 func TestHistoryQueryWorkflowResponse(t *testing.T) {
 	for _, item := range []*types.HistoryQueryWorkflowResponse{nil, &testdata.HistoryQueryWorkflowResponse} {
-		assert.Equal(t, item, ToHistoryQueryWorkflowResponse(FromHistoryQueryWorkflowResponse(item)))
+		roundTripped := ToHistoryQueryWorkflowResponse(FromHistoryQueryWorkflowResponse(item))
+		opt := cmpopts.IgnoreFields(types.MatchingQueryWorkflowResponse{}, "PartitionConfig")
```

Comment:
This not being able to round trip (since the history proto doesn't have a PartitionConfig field) is another argument for not changing the history type.